apps/menu-bar/modules/web-authentication-session/ios/WebAuthContextProvider.swift

```diff
@@ -0,0 +1,13 @@
+import AuthenticationServices
+
+public class WebAuthContextProvider: NSObject, ObservableObject, ASWebAuthenticationPresentationContextProviding {
```

Comment:
Do we need to conform to `ObservableObject` here? We're not using SwiftUI so I don't think it's necessary.

Reply:
You're right, no need to extend ObservableObject